packages/framer-motion/src/components/AnimatePresence/types.ts

```diff
@@ -51,10 +51,25 @@ export interface AnimatePresenceProps {
      * )
      * ```
      *
-     * @beta
+     * @deprecated
+     *
+     * Replace with `mode="popLayout"`
      */
     exitBeforeEnter?: boolean
 
+    /**
+     * Determines how to handle entering and exiting elements.
+     *
+     * - `"sync"`: Default. Elements animate in and out as soon as they're added/removed.
+     * - `"popLayout"`: Exiting elements are "popped" from the page layout, allowing sibling
+     *      elements to immediately occupy their new layouts.
+     * - `"wait"`: Only renders one component at a time. Wait for the exiting component to animate out
+     *      before animating the next component in.
```

Comment:
It would be great to add a link to a code sandbox or similar example, to show the three different types.

Reply:
I'll be putting sandboxes in the documentation but would prefer to reduce the places where I put these links